src/NuGet.Clients/NuGet.VisualStudio.Interop/NuGet.VisualStudio.Interop.csproj

```diff
@@ -4,7 +4,6 @@
   <PropertyGroup>
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
     <Shipping>true</Shipping>
-    <Pack>true</Pack>
```

Comment:
I think it looks unrelated to version. Can we do it in another PR specific to stop packing n.vs.interop ?

Reply:
NuGet.VisualStudio.Interop isn't getting packed, never has. The property we use to determine packing is `PackProject`. This `Pack` property is unused in our build scripts.

I can remove it in another PR, but it's just clarifying the fact that the project isn't being packed, rather than stopping something that is currently packed from being packed.

Reply:
Oh ok, I thought it stopped packing n.vs.interop. I'm OK with this change